regex-base.cabal

```diff
@@ -9,44 +9,55 @@ copyright:              Copyright (c) 2006, Christopher Kuklewicz
 author:                 Christopher Kuklewicz
 maintainer:             hvr@gnu.org
 homepage:               https://wiki.haskell.org/Regular_expressions
-bug-reports:            https://github.com/hvr/regex-base/issues
+bug-reports:            https://github.com/haskell-hvr/regex-base/issues
 synopsis:               Common "Text.Regex.*" API for Regex matching
 category:               Text
 description:
-  This package doesn't provide the ability to do regex matching, but instead provides the type-classes that constitute the abstract API that is implemented by @regex-*@ backends such as
+  This package does not provide the ability to do regex matching,
+  but instead provides the type-classes that constitute the abstract API
+  that is implemented by @regex-*@ backends such as
   .
-  * <//hackage.haskell.org/package/regex-posix regex-posix>
+  * <https://hackage.haskell.org/package/regex-posix regex-posix>
   .
-  * <//hackage.haskell.org/package/regex-parsec regex-parsec>
+  * <https://hackage.haskell.org/package/regex-parsec regex-parsec>
   .
-  * <//hackage.haskell.org/package/regex-dfa regex-dfa>
+  * <https://hackage.haskell.org/package/regex-dfa regex-dfa>
   .
-  * <//hackage.haskell.org/package/regex-tdfa regex-tdfa>
+  * <https://hackage.haskell.org/package/regex-tdfa regex-tdfa>
   .
-  * <//hackage.haskell.org/package/regex-pcre regex-pcre>
+  * <https://hackage.haskell.org/package/regex-pcre regex-pcre>
   .
   See also <https://wiki.haskell.org/Regular_expressions> for more information.
 
 extra-source-files:
   ChangeLog.md
 
 tested-with:
-  GHC ==7.0.2
-   || ==7.2.2
-   || ==7.4.2
-   || ==7.6.3
-   || ==7.8.4
-   || ==7.10.3
-   || ==8.0.2
-   || ==8.2.2
-   || ==8.4.4
-   || ==8.6.5
-   || ==8.8.3
-   || ==8.10.1
+  -- Haskell CI:
+  GHC == 7.0.4
+  GHC == 7.2.2
+  GHC == 7.4.2
+  GHC == 7.6.3
+  GHC == 7.8.4
+  GHC == 7.10.3
+  GHC == 8.0.2
+  GHC == 8.2.2
+  GHC == 8.4.4
+  GHC == 8.6.5
+  GHC == 8.8.4
+  GHC == 8.10.3
+  -- manually (AA, 2021-02-16):
+  GHC == 8.10.4
```

Comment:
These additions break `haskell-ci regenerate`. Maybe have them just as comments

I'm not aware of any tool which reads `tested-with` other than `haskell-ci`.

Reply:
Since compatibility with GHC 9.0 is the main reason for this release, it would be nice to have a test evidence. I'm happy to volunteer adding a `haskell/actions`-based CI job for this, if this is appropriate.

Reply:
@Bodigrim: Happy to get a PR.
The changes for 9.0 were minimal, though: some spaces after `!`s, and only a `base` version bump otherwise.

Reply:
Would you like to test 9.0 only or 8.10.4 as well? Ubuntu only? I was thinking to adapt a subsection of https://github.com/haskell/bytestring/blob/master/.github/workflows/ci.yml

Reply:
@Bodigrim: Anecdotally, Ulf Norell is compiling Agda under GHC 9.0 with a patched version of regex-base.  (Patched to make it compile.)

Reply:
@Bodigrim:
> Would you like to test 9.0 only or 8.10.4 as well? Ubuntu only? I was thinking to adapt a subsection of [haskell/bytestring@`master`/.github/workflows/ci.yml](https://github.com/haskell/bytestring/blob/master/.github/workflows/ci.yml?rgh-link-date=2021-02-19T00%3A01%3A10Z)

Anything that covers the version currently not supplied by `haskell-ci` is fine.  (I understand that you make a separate workflow so the `haskell-ci` workflow can be updated mechanically later.)

Reply:
I trust that @andreasabel tested the releases locally, and I don't think that we need to add transient CI setup to be removed/altered later. It's just waste work.

Reply:
OK, let it be. 
(I'm less optimistic about it being "transient", but we can return to this question in a couple of months)

Reply:
Fair. In a couple of months there will be 9.2